setup.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+from os import path
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+# To use a consistent encoding
+
+BASE_DIR = path.abspath(path.dirname(__file__))
+
+with open(path.join(BASE_DIR, 'README.md')) as f:
+    LONG_DISCRIPTION = f.read()
+
+setup(
+    name='profile-manager',
+
+    version='0.1',
+
+    description='A sample Python project',
+    long_description=LONG_DISCRIPTION,
+
+    # The project's main homepage.
+    url='https://collabora.com/',
+
+    # Author details
+    author='Shane Fagan',
```

Comment:
Well this is just about the packaging info. This one isn't as important as the AUTHORS file (which I haven't added yet)